src/content/docs/en/basics/astro-pages.mdx

```diff
@@ -96,10 +96,18 @@ Files with the `.html` file extension can be placed in the `src/pages/` director
 
 ## Custom 404 Error Page
 
-For a custom 404 error page, you can create a `404.astro` or `404.md` file in `/src/pages`.
+For a custom 404 error page, you can create a `404.astro` or `404.md` file in `src/pages`.
 
 This will build to a `404.html` page. Most [deploy services](/en/guides/deploy/) will find and use it.
 
+## Custom 500 Error Page
+
+For a custom 500 error page to show for pages in your `hybrid` or `server` project that are [rendered on demand](/en/guides/server-side-rendering/#enable-on-demand-server-rendering), create the file `src/pages/500.astro`. This custom page is not available for prerendered pages.
+
+If an error occurs rendering this page, your host's default 500 error page will be shown to your visitor.
+
+During development, if you have a `500.astro`, the error thrown at runtime is logged in your terminal, as opposed to being shown in the error overlay.
```

Comment:
Should we specify since what version this happens?

Reply:
Good call. Throw in a `<Since>` component between those two paragraphs! Check the deploy preview to make sure it looks OK, then fine to merge!

Reply:
```suggestion
<p><Since v="4.10.3" /></p>

During development, if you have a `500.astro`, the error thrown at runtime is logged in your terminal, as opposed to being shown in the error overlay.
```
Why do we need to wrap it inside a <p>?

Reply:
If you can get it to look right without one, go ahead! That's the pattern I use for spacing!